app/services/programme_type_mappings.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class ProgrammeTypeMappings
+  class << self
+    def mappings_enabled?
+      FeatureFlag.active?(:programme_type_changes_2025)
+    end
+
+    def withdrawl_reason(reason:)
+      return reason unless mappings_enabled?
+
+      case reason
+      when "switched-to-school-led"
+        "school-left-fip"
+      else
+        reason
+      end
+    end
+
+    def training_programme(training_programme:)
+      return training_programme.humanize unless mappings_enabled?
+
+      case training_programme
+      when "full_induction_programme", "school_funded_fip"
+        "Provider Led"
+      else
+        "School Led"
+      end
+    end
```

Comment:
`humanize` should be outside of this service class. I'm thinking:
```ruby
    def training_programme(training_programme:)
      return training_programme unless mappings_enabled?

      case training_programme
      when "full_induction_programme", "school_funded_fip"
        "provider_led"
      when "core_induction_programme", "design_our_own"
        "school_led"
      else
        training_programme
      end
    end
```

Reply:
yeah that's a good point, I'll update 👍